records_mover/records/sources/dataframes.py

```diff
@@ -70,6 +71,7 @@ def serialize_dfs(self,
 
         for df in self.dfs:
             with NamedTemporaryFile(prefix='mover_seralized_dataframe') as output_file:
+                df = purge_unnamed_unused_columns(df)
```

Comment:
I hit the need for this in the Postgres integration tests.

Reply:
Fair question.

Locations this occurs:

https://github.com/bluelabsio/records-mover/blob/c0c38af8073590688e79d717910e41bfafbd225f/records_mover/records/targets/table/move_from_dataframes_source.py#L73

* This is while processing things from a dataframes source.

https://github.com/bluelabsio/records-mover/blob/c0c38af8073590688e79d717910e41bfafbd225f/records_mover/records/schema/schema/__init__.py#L130

* This is from a freshly generated dataframe built from our own stream_csv() method.

https://github.com/bluelabsio/records-mover/blob/c0c38af8073590688e79d717910e41bfafbd225f/records_mover/records/sources/dataframes.py#L74

* This is while processing things from a dataframes source and is never turned into a dataframes source.

So nothing here is upstream of each other, no...but perhaps there's an opportunity to get upstream of the current uses:
* Maybe we can have the dataframes source wrap its current iterator with this?
* Maybe the root cause is stream_csv() itself and we can wrap things there?

I can throw this in a backlog item to ponder this a little further if that sounds right to you.